src/actions/const.js

```diff
@@ -2,3 +2,4 @@
 /* Populated by react-webpack-redux:action */
 export const CHANGE_COLOR = 'CHANGE_COLOR';
 export const CHANGE_WIDTH = 'CHANGE_WIDTH';
+export const FINISH_PEN_PREVIEW = 'FINISH_PEN_PREVIEW';
```

Comment:
I don't think we need this action since the action will not be a trigger for other components.
I think, the procedure is like following:
1. Get CHANGE_WIDTH action on PenPreview
2. Show the preview
That's it.

How do you think about?